pkg/adaptor/cloud/libvirt/volume.go

```diff
@@ -107,17 +105,17 @@ func newDefVolumeFromXML(s string) (libvirtxml.StorageVolume, error) {
 	return volumeDef, nil
 }
 
-func timeFromEpoch(str string) time.Time {
-	var s, ns int
+// func timeFromEpoch(str string) time.Time {
+// 	var s, ns int
 
-	ts := strings.Split(str, ".")
-	if len(ts) == 2 {
-		ns, _ = strconv.Atoi(ts[1])
-	}
-	s, _ = strconv.Atoi(ts[0])
+// 	ts := strings.Split(str, ".")
+// 	if len(ts) == 2 {
+// 		ns, _ = strconv.Atoi(ts[1])
+// 	}
+// 	s, _ = strconv.Atoi(ts[0])
 
-	return time.Unix(int64(s), int64(ns))
-}
+// 	return time.Unix(int64(s), int64(ns))
+// }
 
 func uploadVolume(libvirtClient *libvirtClient, volumeDef libvirtxml.StorageVolume, img image) (volumeKey string, err error) {
 
```

Comment:
If we are going with this approach we probably should remove genisoimage from the following files, as well:
```

podvm/Dockerfile.podvm_builder.rhel
podvm/Dockerfile.podvm_builder.centos
Dockerfile
```

Reply:
Dockerfile definitively as it will build the image where the cloud-api-adator service runs. I am not sure about `podvm/Dockerfile.podvm_builder.rhel` and `podvm/Dockerfile.podvm_builder.centos` as  packer might still need `genisoimage` to build the podvm qcow2.